### ISSUE TITLE
More on automated tools and approaches

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-em#116]
One of my concerns about WCAG-EM 1.0 and indeed WCAG 2.x itself, is that there is little to no guidance on use of automated tools. I think there are 3 instances in it in the [Oct 22 Editor's draft](https://w3c.github.io/wai-wcag-em/). 

There are huge problems with automated tools that are well known, but they are also widely adopted by the industry. We should be able to point to any tool leveraging [ACT Rules](https://www.w3.org/WAI/standards-guidelines/act/rules/) in this methodology. Most of us are most familiar with axe, but I totally understand why we wouldn't want to say that here or for that matter Siteimprove (even though [alfa](https://github.com/Siteimprove/alfa) is also an open source ACT based tool). 

Accessibility services are one of the ways that many governments and universities track their accessibility. We know the challenges with [Goodhart's law](https://en.wikipedia.org/wiki/Goodhart%27s_law), but we can also see how this measure can be used to effectively motivate a set of organizations: 
https://analytics.georgia.gov/

There are also now so many tools that simply link together tools (primarily axe) with multi-page/multi-domain crawls. I believe many of the EU's [accessibility monitoring programs](https://accessible-eu-centre.ec.europa.eu/accessibility-monitoring_en) are run with these types of tools. Many of these aren't author facing, but some of them are.

Siteimprove has done some work to integrate their testing suite with the browser through browser extensions. 

There is an unmaintained [Drupal module](https://www.drupal.org/project/accessibility_scanner) which was built to include internal scanning. There is also some great work with [Editoria11y](https://www.drupal.org/project/editoria11y) which has site-wide implications. I don't think this is available in WordPress or Joomla with [Sa11y](https://sa11y.netlify.app/), but there is I think [Joomla includes it by default](https://joomla-projects.github.io/joomla-a11y-checker/). 

It is also just easier to combine testing into other automation, such as what [my colleague wrote about](https://accessibility.civicactions.com/posts/automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe) or for that matter that [GitHub released recently](https://github.com/github/accessibility-scanner).  

I do really love the work done by Singapore's Government Digital Services with [Oobee](https://github.com/GovTechSG/oobee).  I shouldn't be able to find hundreds of accessibility errors in scanning a hundred pages of the W3C.org — I did, though (it is attached). Now there are ways to address this, but I do think we need to address the web at scale. We need to move beyond page by page approaches.

Now having multiple means of collecting information can be a challenge, as there isn't a standardized output that folks agree on. We're working on a solution for that. 

[report.html.zip](https://github.com/user-attachments/files/23061387/report.html.zip)
